routes/web.php

```diff
@@ -18,3 +18,5 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::resource("post", "PostController");
```

Comment:
`Route::post('post', 'PostController@store');` would have been fine here. A resource route has about 5 or 6 routes which you are not implementing.